tidb_binlog/node/node.go

```diff
@@ -1,21 +1,68 @@
 package node
 
 import (
-	pb "github.com/pingcap/tipb/go-binlog"
+	"time"
 )
 
-// Status describes the status information of a tidb-binlog node in etcd
+var (
+	// DefaultRootPath is the root path of the keys stored in etcd.
+	DefaultRootPath = "/tidb_binlog"
```

Comment:
the old path is `/tidb-binlog`, the new version pump and drainer write status to the new path, and can make  tidb  cluster upgrade easy.

Reply:
it's better to organize it like `/tidb-binlog/{version}`